github-dark.css

```diff
@@ -942,7 +942,8 @@
   .ajax-pagination-form .ajax-pagination-btn:hover,
   .ajax-pagination-form .ajax-pagination-btn:focus,
   .manage-repo-access-not-active, .header-search-scope, .reverse-progress-bar,
-  .featurette.py-5, .ace_button:hover {
+  .featurette.py-5, .ace_button:hover,
+  .payment-history .refunded, .payment-history .failed, {
```

Comment:
For future reference, please don't include trailing commas. Luckily I remembered to use the linter and I fixed it before releasing a new update 😉.